### ISSUE TITLE
Markdown toolbar

### ISSUE THREAD

**Issue by robsonsobral** [netlify/netlify-cms#909]
Hi!

I know a lot of devs don't need tools to help them to write markdown, but, for non techie people, a toolbar is a great tool to help them learn markdown.

Is ti possible to add something like the Stackoverflow/stackexchange markdown editor? Is it [Pagedown](https://github.com/StackExchange/pagedown), right? The one [used by Statamic](https://docs.statamic.com/fieldtypes/markdown) is amazing.

**Comment by neutyp** [comment]
Our WYSIWYG("rich text") toolbar icons literally generate Markdown, you just have to toggle to Markdown to see what it generates. Am I misunderstanding what you want? It sounds like we have it already? 

Is there a demo of something that I can look at to consider against ours?

**Comment by robsonsobral** [comment]
Thank you for your attention, @neutyp.

I think the toolbar should stay always enabled. For example, I've coworkers who learned markdown because of statamic toolbar, some years ago. This is why I know that to have a markdown syntax highlighter and a toolbar can be really useful to improve the collaboration between team members; which is exactly what I'm looking for on Netlify-CMS.

It isn't a change good for everybody, I know, but it doesn't get in anybody's way either.

Thank you!

**Comment by tech4him1** [comment]
Agreed. If it wouldn't be too much work, I wouldn't mind having the toolbar be available in MarkDown mode as well.

**Comment by neutyp** [comment]
Ahhh! I get it now. And yes, that seems like a great idea, simply leave the icons active and the toggle would simply toggle how the edits are visualized.

**Comment by tech4him1** [comment]
cc/ @erquhart

**Comment by robsonsobral** [comment]
I'm sorry for have not been clear at the first moment, guys.

**Comment by tech4him1** [comment]
Not a problem, just wanted to clarify 😄.

**Comment by erquhart** [comment]
Agreed on both the toolbar and syntax highlighting.

**Comment by tech4him1** [comment]
Related to #245.

**Comment by tech4him1** [comment]
Related to #245. These can likely be implemented together.

**Comment by erquhart** [comment]
There's a lot going on in that editor, even for devs who do know React. Thanks for the attempt!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by martinjagodic** [comment]
Closing as stale and/or outdated.
